scripts/tests.sh

```diff
@@ -43,8 +57,8 @@ elif [ -z $CI ]; then
 else
     mkdir -p /tmp/test
     # Update hash below if binary is changed.
-    wget -q -O /tmp/test/eth https://github.com/ethereum/cpp-ethereum/releases/download/solidityTester/eth_byzantium2
-    test "$(shasum /tmp/test/eth)" = "4dc3f208475f622be7c8e53bee720e14cd254c6f  /tmp/test/eth"
+    wget -q -O /tmp/test/eth https://github.com/ethereum/cpp-ethereum/releases/download/solidityTester/eth_no_microhttpd
+    test "$(shasum /tmp/test/eth)" = "8e0e2c2b9f4cf4ad8956a81c1520bbb817768a53  /tmp/test/eth"
```

Comment:
Do you want to also update travis to use this newer version? It is fine testing against two different versions though.

Reply:
That's the problem with this PR: The version of some library is incompatible between the two systems.

Reply:
The PR as it is now uses the same version for both.

Reply:
Right, for a second I thought this change is in `circle.yml` :)